include/aspect/particle/manager.h

```diff
@@ -46,7 +46,7 @@
 #include <boost/serialization/unique_ptr.hpp>
 
 #include <random>
-
+#include <aspect/particle/distribution.h>
```

Comment:
please add this further up where the other aspect/particle includes are.

Additionally this now creates a circular include with the distribution.h header (which includes `manager.h`). I fixed this in #6654, which is probably going to be merged before this PR, so it shouldnt pose a problem.